### PR TITLE
RtStat host combo behavior

### LINKED ISSUE #638: rtstat host combo duplicates
**Describe the bug**
rtstat host combo starts showing duplicates after connecting to a host.

**To Reproduce**
1. run rtstat
2. select a host (should look normal)
3. connect
4. select another host
5. see issue

**Expected behavior**
one entry per host/user combination

**Screenshots**
image

**Desktop/ServerSide (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by MikeNeilson**
## Problem Description

Fixes #638. 

Describe the problem you are trying to solve.

## Solution

Clear hostCombo list before elements added.

Future work should create an LrgsConnection object and use a proper model. Also it shouldn't need to constantly reload the file every time the connection changes. That's usually small so I'm not worried about it right now but there's room for improvement.

## how you tested the change

Opened RtStat on system with multiple connections, connected, selected other hosts. Saw that combo box contents stayed static.
if automated tests demonstrating usage are provided in the PR.

## Where the following done:

- [ ] Tests. Check all that apply:
   - [ ] Unit tests created or modified that run during ant test.
   - [ ] Integration tests created or modified that run during integration testing
         (Formerly called regression tests.)
   - [ ] Test procedure descriptions for manual testing
- [ ] Was relevant documentation updated?
- [ ] Were relevant config element (e.g. XML data) updated as appropriate

If you aren't sure leave unchecked and we will help guide you to want needs changing where.

### CODE REVIEW

**Comment by ktarbet** [comment]
I've testing the LRGS Real Time Status UI and I see a change in behavior.

Previously after connecting to a LRGS, the combox box for host, text box for Port, user, password. retain values.
The new version clears all these inputs (except port) after connecting to a LRGS.

**Review by ktarbet — APPROVED** [review]

**Comment by MikeNeilson** [comment]
Discovered both duplicate logic and static cross GUI references to various fields and data. Began process of creating a Dedicated Panel and it's taking, well probably as long as expected.

**Comment by MikeNeilson** [comment]
I have some additional QoL improvements I'm working on, which also exposed a shortcoming in error handling with the changes. So I'll be getting that fixed up and then switch to ready.

**Comment by MikeNeilson** [comment]
Manually verified MessageBrowser still works.

I did want to revise the error handling and will need to make sure that the internationalization on the errors isn't broken, but I'd like to do that in a follow up PR to keep the already growing complexity of this PR down.

File: src/main/java/lrgs/rtstat/hosts/LrgsConnection.java [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+    public static String encryptPassword(String password, String key)
+    {
+        if (password == null || password.isEmpty())
+        {
+            return "";
+        }
+        try
+        {
+            final DesEncrypter de = new DesEncrypter(key);
```
Comment (ktarbet) [inline src/main/java/lrgs/rtstat/hosts/LrgsConnection.java:177]: Could this custom DesEncrypter be swapped out for an Apache or similar alternative library?   Suggest research that in another PR/issue.

File: src/main/java/lrgs/rtstat/RtStat.java [hunk 2]
```diff
@@ -41,7 +44,13 @@ public RtStat(String args[])
 		if (hostname != null)
 		{
 			final String username = cmdLineArgs.getUserName();
-			
+			final String password = cmdLineArgs.getPassword();
+			final int port = cmdLineArgs.getPort();
+			if (username == null || password == null)
+			{
+				log.error("A Username and Password are now required for LRGS connections."
```
[the remark this answers is not in the document]
Comment (MikeNeilson) [reply src/main/java/lrgs/rtstat/RtStat.java:51]: Hmm fair point. I thought I completely broke the code path for that behavior but probably good to leave it in at least for now.

File: src/main/java/lrgs/rtstat/hosts/LrgsConnection.java (hunk 1 above)
Comment (MikeNeilson) [reply src/main/java/lrgs/rtstat/hosts/LrgsConnection.java:177]: Honestly, I'd rather just ditch it entirely, but it is just a wrapper around built-in tools, not a custom DES implementation so probably not worth the trouble.

File: src/main/java/lrgs/rtstat/RtStat.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (MikeNeilson) [reply src/main/java/lrgs/rtstat/RtStat.java]: I've change this to an exception. Passing null as the password would eventually cause the downstream code to fail and an exception is a little more direct.

**Review by ktarbet — APPROVED** [review]

**Pull request merged**